app/views/admin/school_onboardings/email/new.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1>Onboarding: <%= @school_onboarding.school_name %></h1>
+
+<dl>
```

Comment:
This does end up looking a bit small on the page - bootstrap default dl stuff I think